diff --git a/types.d.ts b/types.d.ts
index 8fa5306f239a38b196b93032a27302cf43e42634..4ed28c3a71a7f97a5a92d27c6b068d00258a8df7 100644
--- a/types.d.ts
+++ b/types.d.ts
@@ -8076,7 +8076,7 @@ declare interface JsonGeneratorOptions {
 	 */
 	JSONParse?: boolean;
 }
-type JsonObjectFs = { [index: string]: JsonValueFs } & {
+declare interface JsonObjectFs {
 	[index: string]:
 		| undefined
 		| null
@@ -8085,8 +8085,8 @@ type JsonObjectFs = { [index: string]: JsonValueFs } & {
 		| boolean
 		| JsonObjectFs
 		| JsonValueFs[];
-};
-type JsonObjectTypes = { [index: string]: JsonValueTypes } & {
+}
+declare interface JsonObjectTypes {
 	[index: string]:
 		| undefined
 		| null
@@ -8095,7 +8095,7 @@ type JsonObjectTypes = { [index: string]: JsonValueTypes } & {
 		| boolean
 		| JsonObjectTypes
 		| JsonValueTypes[];
-};
+}
 
 /**
  * Parser options for JSON modules.
