diff --git a/DEPS b/DEPS
index 2923b5ce9..64d6c7fe1 100644
--- a/DEPS
+++ b/DEPS
@@ -136,16 +136,16 @@ deps = {
         "ignore_in_git": True,
         "condition": system in ['linux', 'darwin', 'windows']
     },
-    'buildtools/android_sdk_manager': {
-        "type": "http",
-        "url": {
-            "darwin": "https://dl.google.com/android/repository/commandlinetools-mac-8512546_latest.zip",
-            "linux": "https://dl.google.com/android/repository/commandlinetools-linux-8512546_latest.zip",
-            "windows": "https://dl.google.com/android/repository/commandlinetools-win-8512546_latest.zip"
-        }.get(system, None),
-        "ignore_in_git": True,
-        "condition": system in ['linux', 'darwin', 'windows']
-    },
+    # 'buildtools/android_sdk_manager': {
+    #     "type": "http",
+    #     "url": {
+    #         "darwin": "https://dl.google.com/android/repository/commandlinetools-mac-8512546_latest.zip",
+    #         "linux": "https://dl.google.com/android/repository/commandlinetools-linux-8512546_latest.zip",
+    #         "windows": "https://dl.google.com/android/repository/commandlinetools-win-8512546_latest.zip"
+    #     }.get(system, None),
+    #     "ignore_in_git": True,
+    #     "condition": system in ['linux', 'darwin', 'windows']
+    # },
     'third_party/gyp': {
         "type": "git",
         "url": "https://chromium.googlesource.com/external/gyp",
